src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -177,6 +183,14 @@ class AdditionalDetailsStep extends React.Component {
             errors.addressStreet = true;
         }
 
+        if (_.isEmpty(this.props.walletAdditionalDetailsDraft.addressCity)) {
```

Comment:
Added validation here because we have removed validation for required fields, we can add it again if we want. We only need it for `addressCity`, `addressState` 
let me know if we wanna add it again.

Reply:
@rushatgabhane, need your review on this.